lib/utils/deploy-legacy.js

```diff
@@ -0,0 +1,199 @@
+/*
+ * decaffeinate suggestions:
+ * DS102: Remove unnecessary code created because of implicit returns
+ * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
+ */
+import * as Promise from 'bluebird';
```

Comment:
Removed a `Promise = require('bluebird')` because it's already imported at the top level